src/Service/NixUtils.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Nixify\Service;
+
+use function ord;
+
+use const STR_PAD_LEFT;
+
+final class NixUtils
```

Comment:
I don't think any of the methods on this class need to be instance methods. The class just existed as a namespace for static functions, because PHP doesn't really have a language construct for that.

It's similar to, for example: https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/HttpFoundation/IpUtils.php